### ISSUE TITLE
Improve consistency -  `SV2 or Sv2` and `Stratum V2 or Stratum v2`

### ISSUE THREAD

**Issue by pavlenex** [stratum-mining/stratumprotocol.org#193]
Once we're done shipping a release, would be great to consolidate how we use Stratum v2 or Stratum V2 ; SV2 or Sv2 since we're inconsistent across website and docs currently.

Requires community discussion on what to settle for. I invite everyone to leave comments on their preference.

It seems specifications are inconsistent about this topic as well.

**Comment by pavlenex** [comment]
I usually write Stratum v2 and Sv2, so I'd say I'd go with that.

**Comment by Sjors** [comment]
> I usually write Stratum v2 and Sv2

Same. But maybe that's because I copied it from you :-)

**Comment by pavlenex** [comment]
I guess specs authors call it Stratum V2 and the opening sentence in specs is Stratum V2, from there it's Stratum v2, sv2 and SV2. So it seems Stratum v2 thus SV2 is how it should be.
<img width="601" alt="Screenshot 2024-02-13 at 19 36 36" src="https://github.com/stratum-mining/stratumprotocol.org/assets/36959754/7ab3e8ea-1abe-4210-955d-57ab0334c611">

Again don't really have hard preference as long as we decide as community and then correct inconsistencies 😄

**Comment by xyephy** [comment]
Hi @pavlenex is the final conclusion Stratum V2 and SV2?
If so, can I start on the cleanup?

**Comment by pavlenex** [comment]
Hey @xyephy Correct. Go for it.

**Comment by xyephy** [comment]
@pavlenex  I've exempted links and examples from changes i.e 
```bash
./bitcoin-sv2-tp-0.1.2/bin/bitcoind -sv2 -sv2port=8442 -sv2interval=20 -sv2feedelta=1000 -debug=sv2 -loglevel=sv2:trace
```
(roles/sv2/mining-proxy)

https://github.com/stratum-mining/sv2-spec

Also modified a few Sv1 -> SV1 for uniformity

The fix PR: #209
